Fails to build with debug feature

When I try to build with the debug feature, the build fails with the following errors:

```
$ cargo build --features debug
   Compiling cargo-outdated v0.5.1 (file:///C:/Users/kraaimx/src/cargo-outdated)
error[E0433]: failed to resolve. Maybe a missing `extern crate term;`?
   --> src\macros.rs:17:50
    |
17  |           $config.shell().say(format!($($arg)*), ::term::color::WHITE)?
    |                                                    ^^^^ Maybe a missing `extern crate term;`?
    |
   ::: src\cargo_ops\elaborate_workspace.rs
    |
152 | /         debug!(
153 | |             config,
154 | |             "UPDATE, self: {:?}, key: {:?}, status: {:?}\n",
155 | |             self_pkg.package_id(),
156 | |             pkg_status_key,
157 | |             pkg_status
158 | |         );
    | |__________- in this macro invocation

error[E0433]: failed to resolve. Maybe a missing `extern crate term;`?
   --> src\macros.rs:17:50
    |
17  |           $config.shell().say(format!($($arg)*), ::term::color::WHITE)?
    |                                                    ^^^^ Maybe a missing `extern crate term;`?
    |
   ::: src\cargo_ops\elaborate_workspace.rs
    |
165 | /         debug!(
166 | |             config,
167 | |             "LOOP, parent: {:?}, self: {:?}, compat: {:?}, latest: {:?}\n",
168 | |             parent,
...   |
171 | |             latest_pkg
172 | |         );
    | |__________- in this macro invocation

error[E0433]: failed to resolve. Maybe a missing `extern crate term;`?
   --> src\macros.rs:17:50
    |
17  |           $config.shell().say(format!($($arg)*), ::term::color::WHITE)?
    |                                                    ^^^^ Maybe a missing `extern crate term;`?
    |
   ::: src\cargo_ops\elaborate_workspace.rs
    |
202 | /             debug!(
203 | |                 config,
204 | |                 "NEXT, next_self: {:?}, next_compat: {:?}, next_latest: {:?}\n",
205 | |                 next_self,
206 | |                 next_compat,
207 | |                 next_latest
208 | |             );
    | |______________- in this macro invocation

error[E0433]: failed to resolve. Maybe a missing `extern crate term;`?
   --> src\macros.rs:14:41
    |
14  |         $config.shell().say($message, ::term::color::WHITE)?
    |                                         ^^^^ Maybe a missing `extern crate term;`?
    |
   ::: src\main.rs
    |
241 |     debug!(config, format!("options: {:?}", options));
    |     -------------------------------------------------- in this macro invocation

error[E0599]: no method named `say` found for type `std::cell::RefMut<'_, cargo::core::Shell>` in the current scope
   --> src\macros.rs:14:25
    |
14  |         $config.shell().say($message, ::term::color::WHITE)?
    |                         ^^^
    |
   ::: src\main.rs
    |
241 |     debug!(config, format!("options: {:?}", options));
    |     -------------------------------------------------- in this macro invocation

error[E0599]: no method named `say` found for type `std::cell::RefMut<'_, cargo::core::Shell>` in the current scope
   --> src\macros.rs:17:25
    |
17  |           $config.shell().say(format!($($arg)*), ::term::color::WHITE)?
    |                           ^^^
    |
   ::: src\cargo_ops\elaborate_workspace.rs
    |
152 | /         debug!(
153 | |             config,
154 | |             "UPDATE, self: {:?}, key: {:?}, status: {:?}\n",
155 | |             self_pkg.package_id(),
156 | |             pkg_status_key,
157 | |             pkg_status
158 | |         );
    | |__________- in this macro invocation

error[E0599]: no method named `say` found for type `std::cell::RefMut<'_, cargo::core::Shell>` in the current scope
   --> src\macros.rs:17:25
    |
17  |           $config.shell().say(format!($($arg)*), ::term::color::WHITE)?
    |                           ^^^
    |
   ::: src\cargo_ops\elaborate_workspace.rs
    |
165 | /         debug!(
166 | |             config,
167 | |             "LOOP, parent: {:?}, self: {:?}, compat: {:?}, latest: {:?}\n",
168 | |             parent,
...   |
171 | |             latest_pkg
172 | |         );
    | |__________- in this macro invocation

error[E0599]: no method named `say` found for type `std::cell::RefMut<'_, cargo::core::Shell>` in the current scope
   --> src\macros.rs:17:25
    |
17  |           $config.shell().say(format!($($arg)*), ::term::color::WHITE)?
    |                           ^^^
    |
   ::: src\cargo_ops\elaborate_workspace.rs
    |
202 | /             debug!(
203 | |                 config,
204 | |                 "NEXT, next_self: {:?}, next_compat: {:?}, next_latest: {:?}\n",
205 | |                 next_self,
206 | |                 next_compat,
207 | |                 next_latest
208 | |             );
    | |______________- in this macro invocation

error: aborting due to 8 previous errors

error: Could not compile `cargo-outdated`.

To learn more, run the command again with --verbose.
```

--- a/.travis.yml
+++ b/.travis.yml
@@ -1,7 +1,6 @@
 language: rust
 rust:
   - nightly
-#  - nightly-2016-03-22
   - beta
   - stable
 matrix:
@@ -9,9 +8,8 @@ matrix:
     - rust: nightly
 script:
   - |
-    cargo build &&
-    cargo test
-
+      cargo build
+      cargo build --features debug
 env:
   global:
     secure: JLBlgHY6OEmhJ8woewNJHmuBokTNUv7/WvLkJGV8xk0t6bXBwSU0jNloXwlH7FiQTc4TccX0PumPDD4MrMgxIAVFPmmmlQOCmdpYP4tqZJ8xo189E5zk8lKF5OyaVYCs5SMmFC3cxCsKjfwGIexNu3ck5Uhwe9jI0tqgkgM3URA=

--- a/Cargo.toml
+++ b/Cargo.toml
@@ -1,13 +1,23 @@
 [package]
-authors = ["Kevin K. <kbknapp@gmail.com>", "Frederick Z. <frederick888@tsundere.moe>"]
+authors = [
+    "Kevin K. <kbknapp@gmail.com>",
+    "Frederick Z. <frederick888@tsundere.moe>",
+]
 description = "Cargo subcommand for displaying when dependencies are out of date"
 exclude = ["*.png"]
-keywords = ["cargo", "subcommand", "dependencies", "cargo-subcommand", "deps"]
+keywords = [
+    "cargo",
+    "subcommand",
+    "dependencies",
+    "cargo-subcommand",
+    "deps",
+]
 license = "MIT"
 name = "cargo-outdated"
 readme = "README.md"
 repository = "https://github.com/kbknapp/cargo-outdated.git"
 version = "0.5.1"
+
 [[bin]]
 name = "cargo-outdated"
 
@@ -22,8 +32,12 @@ tabwriter = "~1.0.3"
 tempdir = "~0.3.5"
 toml = "~0.4.3"
 
+[dependencies.termcolor]
+optional = true
+version = "0.3.3"
+
 [features]
-debug = []
+debug = ["termcolor"]
 default = []
 [profile.release]
 lto = true

--- a/src/macros.rs
+++ b/src/macros.rs
@@ -11,10 +11,10 @@ macro_rules! verbose {
 #[cfg(feature = "debug")]
 macro_rules! debug {
     ($config: expr, $message: expr) => (
-        $config.shell().say($message, ::term::color::WHITE)?
+        $config.shell().status_with_color("DEBUG", $message, ::termcolor::Color::White)?
     );
     ($config: expr, $($arg: tt)*) => (
-        $config.shell().say(format!($($arg)*), ::term::color::WHITE)?
+        $config.shell().status_with_color("DEBUG", format!($($arg)*), ::termcolor::Color::White)?
     );
 }
 

--- a/src/main.rs
+++ b/src/main.rs
@@ -30,6 +30,8 @@ extern crate serde;
 extern crate serde_derive;
 extern crate tabwriter;
 extern crate tempdir;
+#[cfg(feature = "debug")]
+extern crate termcolor;
 extern crate toml;
 
 #[macro_use]

